src/main/java/io/voucherify/client/model/loyalties/AddBalance.java

```diff
@@ -15,4 +16,10 @@
 public class AddBalance {
 
   private Long points;
+
+  @JsonProperty("expiration_date")
+  private String expirationDate;
+
+  @JsonProperty("expiration_type")
+  private String expirationType;
```

Comment:
Use BalanceExpirationType here

Reply:
My bad. I missed it. Apologies. It's done now.